app/domain/authentication/authn_k8s/k8s_object_lookup.rb

```diff
@@ -36,14 +36,13 @@ def bearer_token
       end
 
       def ca_cert
-        cert = K8sContextValue.get(
+        K8sContextValue.get(
           @webservice,
           SERVICEACCOUNT_CA_PATH,
           VARIABLE_CA_CERT
-        )
-
-        raise Err::MissingCertificate if cert.blank?
-        cert
+        ).tap do |cert|
```

Comment:
This also seemed more readable before the addition of `tap`

Reply:
i'll address all the taps in another PR once this one is merged as we have a lot of them throughout the code